DataStructure/Class1.cs

```diff
@@ -1,26 +1,15 @@
-﻿using System;
-
-namespace DataStructures.LinkedLists
+﻿namespace DataStructures.LinkedLists
 {
-    public class LinkedList

```

Comment:
This change is wrong.  You need to have your Linked List class, and then you need to have your Node class separately that your LinkedList class uses to make new nodes for the list.  Go to Keith's demo code to see how he has his projects setup.  Also, look through all of his code to see what he has that you are missing throughout your files (ie. usings at the top, classes in certain places, and so on).
https://github.com/dahlbyk-demo/cr-dotnet-401d2-dsa/tree/b54b2d00d4f09b1319c859e07cc2285ba05f4438